ci-operator/jobs/openshift/csi-driver-shared-resource-operator/openshift-csi-driver-shared-resource-operator-release-4.10-presubmits.yaml

```diff
@@ -1,7 +1,7 @@
 presubmits:
   openshift/csi-driver-shared-resource-operator:
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
is this needed for the skip thing ?

Ideally the semantic is "aways run but skip if only changed ..."

Reply:
Yes, it is needed.
That change was done when running "make update" due to the skip_if_only_changed in the configuration files.

Reply:
ok thanks for that confirmation

and we  get the "aways run but skip if only changed ..." semantic, right ?

Reply:
correct